Dockerfile

```diff
@@ -15,7 +16,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     # install new versions of these packages backported to Debian stable;
     # Debian does not add new features or break backwards compatibility within
     # a stable release, but for these dependencies we need versions that do.
-    && apt-get install -y --no-install-recommends -t jessie-backports \
+    && apt-get install -y --no-install-recommends -t stretch-backports \
         maven \
         openjdk-8-jdk \
```

Comment:
These packages do not really need to be backported any longer now—stretch already has the newer version we require in the main repository. I'd suggest just removing the extra logic it like I did in https://github.com/thehyve/cbioportal-docker/commit/04ceb41616071b24324aaf27038e59608c0b185e.

Reply:
Agree! Will be pushing a more simplified version to the same branch shortly